components/core/CMakeLists.txt

```
@@ -21,6 +21,15 @@ if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     set(CMAKE_BUILD_TYPE "${default_build_type}" CACHE STRING "Choose the type of build." FORCE)
 endif()
 
+if(CMAKE_BUILD_TYPE MATCHES "Release")
+    include(CheckIPOSupported)
+    check_ipo_supported(RESULT IPO_SUPPORTED OUTPUT IPO_CHECK_MSG)
```

```suggestion
    check_ipo_supported(RESULT IPO_SUPPORTED OUTPUT)
```

Since we don't use `IPO_CHECK_MSG` any more.